docs/semgrep-app/integrations.md

```diff
@@ -8,12 +8,97 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrations
```

Comment:
Thanks for pointing out the vagueness. We are integrating **with** third party apps, not just "tools." We are not just setting up notifications, because some integrations create tickets within the app. I will update these last 4 findings tomorrow.

Thanks for all the help, @adamkvitek !